dev-requirements.txt

```
@@ -1,7 +1,7 @@
 # install latest changes in dbt-core
 # TODO: how to automate switching from develop to version branches?
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
+git+https://github.com/dbt-labs/dbt-core.git@split-part-negative-part-number#egg=dbt-core&subdirectory=core
+git+https://github.com/dbt-labs/dbt-core.git@split-part-negative-part-number#egg=dbt-tests-adapter&subdirectory=tests/adapter
```

Restore original dev-requirements.txt after merge of https://github.com/dbt-labs/dbt-core/pull/7200
```suggestion
git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
```